### ISSUE TITLE
Could not load plugin mesh defined in seneca-mesh as a require call inside the plugin

### ISSUE THREAD

**Issue by mmick66** [senecajs/seneca-mesh#71]
I am trying to install Seneca mesh but I get an error:

Seneca Fatal Error
\==================
Message: seneca: use-plugin: Could not load plugin mesh defined in seneca-mesh as a require call inside the plugin (or a module required by the plugin) failed: Error: Cannot find module '/root/gmh-backend-solution/node_modules/msgpack/lib/../build/Release/msgpackBinding' at Function._load (/usr/local/lib/node_modules/pm2/node_modules/pmx/lib/transaction.js:62:21) at require (internal/module.js:20:19) at Object.<anonymous> (/root/gmh-backend-solution/node_modules/msgpack/lib/msgpack.js:6:14).

Needless to say that the mesh is not starting at all...

**Comment by thatisuday** [comment]
@mmick66 Same error! Did u fix it?

**Comment by rjrodger** [comment]
It's probably missing seneca-balance-client - docs need to clearer on this.

Also Seneca plugin `depends` should indicate this.

**Comment by rrrodzilla** [comment]
Same error here. Definitely have installed (multiple times) seneca-balance-client as well.  Developing in Ubuntu on Windows 10.

**Comment by rrrodzilla** [comment]
Ahhhhh resolved...@rjrodger, maybe we can update the docs to include this tidbit of info for developers on Windows working via the Windows Linux Subsystem (WSL)?  (there's a lot of us)
After verifying that the Windows build tools were installed (and reinstalling with):
`npm install --global --production windows-build-tools --vs2015` in an elevated Powershell window, I went back to linux and when i ran the following command in Ubuntu:
`make --version`
it was returning a message saying there was a missing makefile so I installed make with:
`sudo apt-get install make`
then calling --version on make returned a valid response.
This, unfortunately only got me incremental progress as I was still getting a make error when trying to install seneca-mesh.
Checking the version of g++ was returning an invalid response so then I installed it with:
`sudo apt install g++`
I closed and reopened my Ubuntu session and seneca-mesh is now installed successfully and running my service.
